### DISCUSSION TITLE
Ingress message rate are blank in management UI after an upgrade, other metrics are intact

### DISCUSSION THREAD

**Discussion by lakshyasharma1196** [rabbitmq/rabbitmq-server#11735] [Questions]
Hi Team,
I m upgrading from 3.11 to 3.13. Currently 3.13 is in load testing phase. I m noticing why incoming table on queue is showing blank.
<img width="1055" alt="image" src="https://github.com/user-attachments/assets/889c6d1e-2307-4340-8e83-9f3c9cddda70">
<img width="1274" alt="image" src="https://github.com/user-attachments/assets/871b0314-d59f-4aeb-8ca4-d5db9ed272ee">

**Comment by lukebakken** [comment]
I just started up a RabbitMQ cluster using [my project](https://github.com/lukebakken/docker-rabbitmq-cluster/tree/main) and everything works fine:

![image](https://github.com/user-attachments/assets/c52cd920-dc77-46d3-a65b-d976234ebd1c)

You will have to provide more information:
* Why in the world are you using a 13-node cluster?
* How _exactly_ are you upgrading?
* Have you FORCE-refreshed your browser window?

**Reply by lakshyasharma1196** [reply]
I m still unable to get data in incoming column with my 5 node cluster. Above 13 nodes cluster was for load testing purpose only.
Could you please help if I m missing something while spinning up the cluster. 

<img width="1161" alt="image" src="https://github.com/user-attachments/assets/57aaafea-0888-49b3-9674-f95b465639c8">


Publish graph is also missing here:
<img width="1173" alt="image" src="https://github.com/user-attachments/assets/a758f1f8-4880-4775-8355-60db7b3af309">

**Reply by michaelklishin** [reply]
We do not guess in this community. You haven't provided any information for us to work with: no logs, no details of the test, no details on whether the [Prometheus scraping endpoint](https://www.rabbitmq.com/docs/prometheus) returns the metric, nothing.

Since this is after a upgrade that jumps two series, consider clearing browser cache as mentioned in the guide on Upgrades.

You can also reset the stats database using buttons on the list of cluster members, for one node or all of them.

**Reply by lakshyasharma1196** [reply]
Here are details:
Rabbitmq service
<img width="1434" alt="image" src="https://github.com/user-attachments/assets/996c5d5d-0e48-4ca7-bddc-cddaea41cf77">

Enabled Plugins
<img width="448" alt="image" src="https://github.com/user-attachments/assets/2800a6cb-5b7b-45e5-aa0a-bc5cb0505ea4">
Incoming messages metrics
<img width="978" alt="image" src="https://github.com/user-attachments/assets/663c2593-8709-4fc0-8f84-e006cefe7792">
Please let me know which details are required. I will share it
Thanks in advance !

**Reply by michaelklishin** [reply]
Start with upgrading to `3.13.6`. Besides the fact that we won't investigate any behavior of outdated patch releases, this will involve a rolling restart of all nodes. If that does not help, I will explain how to dump relevant metrics tables using `rabbitmqctl eval`.
